moja-kuhinjica-frontend/src/components/input/FormInput.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from './FormInput.module.scss'
+import Image from 'next/image'
+
+interface IFormInputProps {
+    src: string
+    content: string
+    type: string
+}
+
+export const FormInput = (props: IFormInputProps) => {
```

Comment:
If you have added type props, you should also deconstruct props `{src, content, type}`

Reply:
Done.